### ISSUE TITLE
Error with wp-cli

### ISSUE THREAD

**Issue by AurelienCepede** [soflyy/oxygen-bugs-and-features#3310]
I just installed the new version of oxygen and try to use wp-cli and got this error :
```
PHP Fatal error:  Uncaught Error: Call to a member function api_request() on null in /var/www/html/wp-content/plugins/oxygen/component-framework/admin/updater/edd-updater.php:258
Stack trace:
#0 /var/www/html/wp-includes/class-wp-hook.php(308): WP_CLI\Runner->{closure}(false, Array, 'oxygen/function...', Array)
#1 /var/www/html/wp-includes/plugin.php(205): WP_Hook->apply_filters(false, Array)
#2 /var/www/html/wp-includes/update.php(511): apply_filters('update_plugins_...', false, Array, 'oxygen/function...', Array)
#3 [internal function]: wp_update_plugins()
#4 phar:///bin/wp/vendor/wp-cli/extension-command/src/WP_CLI/CommandWithUpgrade.php(492): call_user_func('wp_update_plugi...')
#5 phar:///bin/wp/vendor/wp-cli/extension-command/src/Plugin_Command.php(1175): WP_CLI\CommandWithUpgrade->_list(Array, Array)
#6 [internal function]: Plugin_Command->list_(Array, Array)
#7 phar:///bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Dispatcher/CommandFactory.php(100): call_user_func(Array, Array, Array)
#8 [internal function]: WP_CLI\Dispatcher\CommandFactory::WP_CLI\Dispatcher\{closure}(Array, Array)
#9 phar:///bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Dispatcher/Subcommand.php(491): call_user_func(Object(Closure), Array, Array)
#10 phar:///bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php(419): WP_CLI\Dispatcher\Subcommand->invoke(Array, Array, Array)
#11 phar:///bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php(442): WP_CLI\Runner->run_command(Array, Array)
#12 phar:///bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php(1256): WP_CLI\Runner->run_command_and_exit()
#13 phar:///bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Bootstrap/LaunchRunner.php(28): WP_CLI\Runner->start()
#14 phar:///bin/wp/vendor/wp-cli/wp-cli/php/bootstrap.php(78): WP_CLI\Bootstrap\LaunchRunner->process(Object(WP_CLI\Bootstrap\BootstrapState))
#15 phar:///bin/wp/vendor/wp-cli/wp-cli/php/wp-cli.php(32): WP_CLI\bootstrap()
#16 phar:///bin/wp/php/boot-phar.php(11): include('phar:///bin/wp/...')
#17 /bin/wp(4): include('phar:///bin/wp/...')
#18 {main}
  thrown in /var/www/html/wp-content/plugins/oxygen/component-framework/admin/updater/edd-updater.php on line 258
```
When I go to the file and var_dump the `$oxygen_updater` I get this :
```
/var/www/html/wp-content/plugins/oxygen/component-framework/admin/updater/edd-updater.php:258:
class OxygenMainPluginUpdater#2171 (6) {
  public $oxygen_url =>
  string(25) "https://oxygenbuilder.com"
  public $edd_updater =>
  NULL
  public $prefix =>
  string(7) "oxygen_"
  public $plugin_name =>
  string(6) "Oxygen"
  public $priority =>
  int(5)
  public $license_text =>
  string(37) "Enter your license key to get updates"
}
```

So I change the line 258 by :
```
$version_info = $oxygen_updater->edd_updater && $oxygen_updater->edd_updater->api_request( 'plugin_latest_version', array( 'slug' => 'functions' ) );
```
Maybe there is a more cleanable way to solve this issue ?

Many thanks !
Oxygen is beautiful and make the world better ❤️

**Comment by Spellhammer** [comment]
This is resolved in Oxygen 4.6.1 RC 2, which you can get at https://oxygenbuilder.com/portal.

**Comment by AurelienCepede** [comment]
Awesome !
